plutus-playground-server/src/Playground/Interpreter.hs

```diff
@@ -166,6 +175,7 @@ runghcOpts =
     , "-XTemplateHaskell"
     , "-XScopedTypeVariables"
     , "-O0"
+    -- , "-package plutus-tx"
```

Comment:
was that not re-enabled recently?

Reply:
I'm waiting to see if it's enough to use the Stack nix integration.

Reply:
It's commented out rather than deleted so no change to functionality, it's just easier to switch on and off until the issue is sorted permanently. I will test with Stack nix integration now.